packages/ProgressIndicators/src/ProgressBar/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+/** Props for ProgressBar */
+interface ProgressBarProps {
+  animate: boolean /** apply animation to progressBar */;
+  decimalPoints: number /** Rounds off the value to this number */;
+  height: string /** resizes the height of the progressBar */;
+  min: number /** set lower bound for the progressBar range */;
+  max: number /** set upper bound for the progressBar range */;
+  progressStrippedClass: string /** set progressBar class to stripped */;
+  progressAnimationClass: string /** set progressBar class to Animate */;
+  value: number /** Represents the progress bar value */;
+  lineColor: string /** set line colors */;
+  lineColorThresholds?: { [key: string]: number } | undefined /** set linecolor threshold */;
```

Comment:
what would happen if we removed "undefined" here?  I think the `?` means undefined is implied